react/react1/week3/vite-project/src/create.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+const Create = () => {
+  const [title, setTitle] = useState("");
```

Comment:
We could have just made a single state "todo" being an object with these variables